### DISCUSSION TITLE
在 pnpm 创建的 monorepo 仓库中如何给子一级的 vue3 - vite 项目添加 naive-ui 的 ts 类型定义

### DISCUSSION THREAD

**Discussion by wewindy** [tusen-ai/naive-ui#5732] [Q&A] [answered]
我使用 pnpm 创建了一个 monorepo，其中有一个工程是 vite5+vue3+ts5 创建的页面项目，我在这个页面项目中添加了 naive-ui 依赖，并在此项目下的 tsconfig.json 中，按照官网的教程添加了 `"types": "naive-ui/volar"`，且 Volar 插件已处于 take-over 模式。

虽然按需导入 naive-ui 组件可以正常工作，但是不具备类型提示使得开发还是比较难受的，来问问有无遇到一样问题的朋友

<img width="660" alt="image" src="https://github.com/tusen-ai/naive-ui/assets/26168955/98917606-b05f-45eb-b57b-7a4a83ed7935">

<img width="393" alt="image" src="https://github.com/tusen-ai/naive-ui/assets/26168955/73f117af-9692-4564-82de-5c171bfff820">

**Comment by jahnli** [comment, marked as answer]
考虑可能是 `volar` 的问题
